app/models/student_loans/eligibility.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module StudentLoans
+  class Eligibility < ApplicationRecord
+    self.table_name = "student_loans_eligibilities"
+
+    enum qts_award_year: {
+      "before_2013": 0,
+      "2013_2014": 1,
+      "2014_2015": 2,
+      "2015_2016": 3,
+      "2016_2017": 4,
+      "2017_2018": 5,
+      "2018_2019": 6,
+      "2019_2020": 7,
+    }, _prefix: :awarded_qualified_status
+
+    validates :qts_award_year, on: [:"qts-year"], presence: {message: "Select the academic year you were awarded qualified teacher status"}
```

Comment:
Possibly obvious Rails question, but when does this validator get called with a context?

Reply:
The `ClaimUpdate` class [updates the claim in current context](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/app/models/claim_update.rb#L43), which gets called by the [controller](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/app/controllers/claims_controller.rb#L44)

Reply:
Ah ok, and that context gets passed through to child models?

Reply:
Yep, that's right.

Reply:
To the best of my knowledge custom validators have to be called explicitally with either `valid?(:context)`, 'invalid?(:context)` or `save(:context)`.

That makes our calls on line 43 in `claim_update.rb` and line 121 in `tslr_claim.rb` models.

https://guides.rubyonrails.org/active_record_validations.html#on

Looking at line 121 in the tslr claim model makes me think you are right about needing to validtate on `:submit`, feels like something we should have test coverage for but I am struggling to find any, I'll pick this up with @tekin.